### ISSUE TITLE
Moving from adal and requiring admin approval now

### ISSUE THREAD

**Issue by jeffj6123** [AzureAD/microsoft-authentication-library-for-js#3632]
### Core Library

ADAL.js (adal.js)

### Core Library Version

1.0.18

### Wrapper Library

ADAL Angular (adal-angular)

### Wrapper Library Version

1.0.18

### Description

Having upgraded an angularjs application to angular 11, we previously were using adal to handle authentication.
I wanted to move to msal but it appears there are additional permissions being requested now.
This isnt ideal because this would require all users of this product to update their AAD settings which previously didnt have this issue.
(below being the blocker from just swapping over the libraries with credentials crossed out)
shouldnt the default permissions not require admin consent
https://docs.microsoft.com/en-us/graph/permissions-reference#openid-permissions ?
![permissions](https://user-images.githubusercontent.com/15344718/118041076-376e3d00-b327-11eb-8f51-12a8d54128fb.PNG)

The flow currently is,
On application load, inside app_initializer we grab the AAD information for the user's configuration(The application is served by the product) and then previously would create the authentication context and redirect without any additional permissions required.

To summarize:
We only really need 1 piece of information, the role configured with the user in the tenant but we also show the name.
From my understanding adal used the V1, endpoint and MSAL uses the V2 endpoint for authentication. Is that why there are additional permissions required for a user to login?

For reference here is context on what would previously be configured
https://docs.microsoft.com/en-us/azure/service-fabric/service-fabric-cluster-creation-setup-aad

### Identity Provider

Azure AD / MSA

### Source

Internal (Microsoft)

**Comment by jo-arroyo** [comment]
@jeffj6123 thanks for your question. We're currently investigating this and will get back to you with a response.

**Comment by jeffj6123** [comment]
@jo-arroyo after talking around my team trying to get more information I was pointed out to this In our docs here https://docs.microsoft.com/en-us/azure/service-fabric/service-fabric-cluster-creation-setup-aad#create-azure-ad-applications-and-assign-users-to-roles
Specifically
"You are prompted to sign in to an account that has administrative privileges for the Azure AD tenant. After you sign in, the script creates the web and native applications to represent your Service Fabric cluster. If you look at the tenant's applications in the Azure portal, you should see two new entries"

When switching to msal from adal, would it be expected that  new web/native applications are being created? Or should it be able to use the same ones as before?

**Comment by jo-arroyo** [comment]
@jeffj6123 It does appear that the setup scripts in the Service Fabric docs you linked are creating new apps, so yes, that could require consent. On our side, having the same app upgrade should not usually need additional consent, but perhaps this is a specific scenario that does. This doesn't seem to be something MSAL can help with, please contact the Service Fabric team or Azure Support.
